segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/Metrics.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage.impl.rocksdb;
+
+import io.pravega.shared.MetricsNames;
+import io.pravega.shared.metrics.MetricsProvider;
+import io.pravega.shared.metrics.OpStatsLogger;
+import io.pravega.shared.metrics.StatsLogger;
+
+/**
+ * Metrics for RocksDB.
+ */
+final class Metrics {
```

Comment:
Better to qualify `Metrics` as in the other comment above.

Reply:
Even though I'm just following the pattern set elsewhere, I am going to fix this here and everywhere else, since you both are asking me.